core/src/test/scala/org/apache/spark/util/AkkaUtilsSuite.scala

```diff
@@ -27,6 +29,8 @@ import org.apache.spark._
 import org.apache.spark.scheduler.MapStatus
 import org.apache.spark.storage.BlockManagerId
 
+import SSLSampleConfigs._
```

Comment:
Can you change this to `import org.apache.spark.SSLSampleConfigs._` and place it with the other imports?